### PR TITLE
fix: Docker build fails

### PR DESCRIPTION
**Pull request by FelixTing**
- Add required alpine packages to build-stage.
- For the module name and docker image tag, rename "IOTechSystems" and "iotechsystems" to "edgexfoundry".

fix: #4 

Signed-off-by: Felix Ting <felix@iotechsys.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/IOTechSystems/device-usb-camera
+module github.com/edgexfoundry/device-usb-camera
 
 go 1.16
```
Comment (cloudxxx8) [inline go.mod:3]: use go 1.17

File: Dockerfile [hunk 2]
```diff
@@ -19,7 +19,7 @@ FROM ${BASE} AS builder
```
Comment (cloudxxx8) [inline Dockerfile:19]: use golang:1.17-alpine3.15

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]
build the image from `alpine:3.15`

File: go.mod (hunk 1 above)
Comment (FelixTing) [reply go.mod]: modified

File: Dockerfile (hunk 2 above)
Comment (FelixTing) [reply Dockerfile:19]: modified

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**